docs/GDB/Allocations/AllocateResourcefromGDB.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Resource Allocation from Graphical Dispatch Board"
+parent: "GDB Allocations"
```

Comment:
Should this not be "Allocations"? The title of the index.md is "Allocations", not "GDB Allocations" so it won't show in the correct anchor points that way.